plugins/dominant-color-images/readme.txt

```diff
@@ -49,6 +49,12 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 1.1.2 =
+
+**Bug Fixes**
+
+* Re-remove unneeded phpcs:ignore. ([1231](https://github.com/WordPress/performance/pull/1231))
```

Comment:
Is that really worth doing a release for? 😄

Reply:
Hah, right. I guess I'll revert the changes to the versions so that no new release is made for this plugin.

Reply:
Done in 701c1c7c9822f59aa321d3e40e8ad879c5d6d1b8